packages/cli/generators/openapi/utils.js

```diff
@@ -148,6 +153,9 @@ function escapeIdentifier(name) {
   if (!name.match(SAFE_IDENTIFER)) {
     name = _.camelCase(name);
   }
+  if (DECORATOR_NAMES.includes(name)) {
```

Comment:
Are these changes related to the escape comment fix? It might be good to separate them into a different commit if not.

Reply:
Refactored into two commits.